app/controllers/ChooseAnOptionController.scala

```diff
@@ -85,7 +85,7 @@ class ChooseAnOptionController @Inject()(resultsFileConnector: ResidencyStatusAP
     } else {
       Messages("yesterday")
     }
-    Messages("formatted.upload.timestamp", todayOrYesterday, uploadDate.toString("H:mm"))
+    Messages("formatted.upload.timestamp", todayOrYesterday, uploadDate.toString("H:mma").toLowerCase())
```

Comment:
This is partly correct, in Joda DateTime the H is the hour of day in the format of hour of day (0~23). This should be changed to h:mma. This page(https://www.joda.org/joda-time/key_format.html) shows the different patterns for formatting DateTimes